cypress/integration/fetch_items.js

```diff
@@ -1,37 +1,48 @@
-describe("Show list of items", function() {
```

Comment:
Might not be a bad idea to do a separate pull request after we merge and switch all of our tests to using single instead of double quotes for consistency :)

Reply:
I think the prettier plugin will do that for us. Is that right @alejandronanez?